kbase-extension/static/kbase/js/widgets/narrative_core/upload/stagingAreaViewer.js

```diff
@@ -600,37 +675,101 @@ define([
                 );
         },
 
+        renderImportButton: function() {
+
+            let importButton = $('<button></button>')
+            .addClass('kb-staging-table-import__button btn btn-xs btn-primary')
+            .text('Import Selected');
```

Comment:
very minor, but does prettier need to standardise the formatting of this file?

Reply:
i haven't run any linters/formatters on it yet - but can do that ahead of final submission